src/api/browserview/browserview.ts

```diff
@@ -41,7 +41,17 @@ export interface BrowserViewCreationOptions extends BrowserViewOptions {
     };
 }
 
+/**
+ * @lends View
+ */
 export class BrowserViewModule extends Base {
+    /**
+     * Creates a new View.
+     * @param { View~options } options - View creation options
+     * @return {Promise.<View>}
```

Comment:
View => BrowserView?

Reply:
Since we know for sure we're renaming it to View, I just added it the right way here so we don't have to change it later